tokens/tsm-report_test.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2025 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+package tokens
+
+import (
+	"github.com/stretchr/testify/assert"
+	"reflect"
+	"testing"
+)
+
+var provider = "sev_guest"
+var serviceProvider = "svsm"
+var outblob = []byte{117, 24, 74, 133, 21, 198, 189, 177, 81, 18, 129, 84}
+var auxblob = []byte{120, 112, 34, 102, 228, 162, 111, 217, 232, 169, 167, 213, 197, 209, 94, 121, 161, 231, 170, 64}
+
+func Test_TSMReport_Valid_Pass(t *testing.T) {
+	report := &TSMReport{
+		Provider:        provider,
+		OutBlob:         outblob,
+		AuxBlob:         auxblob,
+		ServiceProvider: &serviceProvider,
+		ManifestBlob:    []byte{170, 13, 189, 115, 132, 249, 168, 13, 253, 229, 76, 198},
```

Comment:
this seems to be reused as-is in L33.  It could be made into a variable like `outblob` and `auxblob`.